test/test_geo.py

```diff
@@ -281,7 +282,7 @@ class TestBlock(unittest.TestCase):
                  "still too much points":
                      [(63, 30), (63, 31), (63, 32), (63, 33)],
                  "point missing: 9":
-                     [(1, 1),         (1, 3),
```

Comment:
I would prefer the previous version of whitespace for those two tests.

Reply:
Sorry, my IDE decided it wanted to reformat those and I didn't have the heart to say no. As Dumbledore said, "It takes a great deal of bravery to stand up to our enemies, but just as much to stand up to our friends." Fixed in c1882da86de3efb31ff395d60c4b08a512c4eed7.